tests/std/tests/P0339R6_polymorphic_allocator/test.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <memory_resource>
+
+using std::pmr::polymorphic_allocator;
+
+void allocate_bytes_test() {
+    constexpr int N = 5;
+
+    polymorphic_allocator<> alloc{};
+
+    void* vp = alloc.allocate_bytes(sizeof(int) * N, alignof(int));

```

Comment:
This test is good; should there also be test coverage for the default function argument of `alignment = alignof(max_align_t)`?